app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line" : error["line_number"],
+        "column" : error["column_number"],
+        "message" : error["text"],
+        "name" : error["code"],
+        "source" : "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors" : [],
+        "path" : file_path,
+        "status" : "passed"
+    } if errors == [] else {
+        "errors" : [
+            format_linter_error(error)
+            for error in errors
+        ],
+        "path" : file_path,
+        "status" : "failed"
+    }
```

Comment:
Please, use if condition in a value for `errors` key instead of forming two dicts here

Reply:
But statuses at these dicts are different too. OR if double "if" condition like this:

def format_single_linter_file(file_path: str, errors: list) -> dict:
    return {
        "errors" : [] if errors == [] else
        [
            format_linter_error(error)
            for error in errors
        ],
        "path" : file_path,
        "status" : "passed" if errors == [] else"failed"
    }
    
    is possible?